### ISSUE TITLE
Can't open Introducing the Column Model.ipynb

### ISSUE THREAD

**Issue by apendergrass** [brian-rose/climlab#11]
I've been able to open and run most of the ipython notebooks in courseware, but one doesn't work:  Introducing the Column Model.ipynb. 
```
Angies-MacBook-Pro:courseware persmac$ ipython notebook Introducing\ the\ Column\ Model.ipynb 
[I 13:35:53.128 NotebookApp] The port 8888 is already in use, trying another random port.
[I 13:35:53.134 NotebookApp] Serving notebooks from local directory: /Users/persmac/Documents/climlab/courseware
[I 13:35:53.134 NotebookApp] 0 active kernels 
[I 13:35:53.134 NotebookApp] The Jupyter Notebook is running at: http://localhost:8889/
[I 13:35:53.134 NotebookApp] Use Control-C to stop this server and shut down all kernels (twice to skip confirmation).
[W 13:35:54.551 NotebookApp] Unreadable Notebook: /Users/persmac/Documents/climlab/courseware/Introducing the Column Model.ipynb NotJSONError('Notebook does not appear to be JSON: u\'{\\n "metadata": {\\n  "name": "",\\n  "s...',)
[W 13:35:54.551 NotebookApp] 400 GET /api/contents/Introducing%20the%20Column%20Model.ipynb?type=notebook&_=1456864554015 (::1) 3.01ms referer=http://localhost:8889/notebooks/Introducing%20the%20Column%20Model.ipynb
```

I'm not sure what that error means. Any thoughts?

**Comment by brian-rose** [comment]
Yeah, that one notebook got corrupted at some point. I should be able to rescue it from earlier versions. Will try to get to that soon. 

All the other notebooks should work out of the box.

**Comment by apendergrass** [comment]
I think they do!

**Comment by brian-rose** [comment]
This is now fixed in the latest commit aebbe14b637a7dee551e81461959222c089b5717

Although the notebook is rather out of date. 

There are better examples of working with column models in this repo: https://github.com/brian-rose/ClimateModeling_courseware
